.github/workflows/coverage.yaml

```diff
@@ -1,10 +1,8 @@
-name: Build
+name: Coverage check
 
 on:
   pull_request:
     branches: [ v1.* ]
-  push:
-    branches: [ v1.*]
```

Comment:
This will make it not run on v1.21 , which is required for codecov to actually display our current coverage and coverage over time